apiserver/paasng/paas_wl/workloads/processes/entities.py

```diff
@@ -202,6 +202,10 @@ def main_container_name(self) -> str:
             return f"{self.app.region}-{self.app.scheduler_safe_name}"
         return self.type
 
+    @property
+    def available_instance_count(self):
+        return len([instance for instance in self.instances if instance.ready and instance.version == self.version])
```

Comment:
Nit: `return sum(1 for instance in self.instances if instance.ready and instance.version == self.version)`